src/Token.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TOKEN_H
+#define TOKEN_H
+#include "TokenType.h"
+#include <any>
+#include <string>
+namespace Tolkien {
```

Comment:
fumasti
```suggestion
namespace loxs {
```

:octocat: